CONTRIBUTING.md

```diff
@@ -550,6 +550,239 @@ cat generated/watcher/watcher.log  # See k_ids field for each core in the last d
   ensure that your commit message follows our required format as outlined above
   in this section. Failure to do so is a violation of our standards.
 
+### Git commit runbook
+
+The tt-metal repository has a single main branch, with pull requests being
+merged in via rebase.
+
+For a pull request, you will need to deal with a local branch and an origin
+branch.
+
+The local branch should be on your VM, and you should interact with it through
```

Comment:
I do treat contributing guide as a public guide, applicable to oss contributors, who run on their real machines.

Reply:
True. It doesn't matter whether the machine is virtual or not. Updated.